packages/sanity/src/core/store/_legacy/document/document-pair/checkoutPair.ts

```diff
@@ -120,13 +128,16 @@ function toActions(idPair: IdPair, mutationParams: Mutation['params']) {
   })
 }
 
-function serverCommitMutations(
+function commitActions(
```

Comment:
With this liveEdit change, it's become clear that `commitMutations` is going to stay, and has a different meaning. I've changed this function name to better reflect what it does (rather than being a temporary intermediary)